CRM/Contribute/BAO/Contribution.php

```diff
@@ -1316,68 +1316,11 @@ public static function sortName($id) {
    * @return array
```

Comment:
Seeing as we're in the headspace - please could you add a helpful comment into annual function's header? What is annual? What does it do?

Reply:
done